doc/source/ref/signal-extension-modes.rst

```diff
@@ -90,7 +90,6 @@ computations can be performed with the `periodization`_ mode:
 **Example:**
 
 .. try_examples::
-  :button_text: Try it in your browser!
```

Comment:
@agriyakhetarpal on this page the buttons disappeared. Maybe something went wrong here, could you check please?

Reply:
That's quite strange. I'm able to see the two buttons on https://pywavelets.readthedocs.io/en/latest/ref/signal-extension-modes.html, which is mapped to this file, and I verified that this file locally also contains two buttons. Could you please try with a fresh browser instance, or reload this page with caches disabled (<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>R</kbd>), perhaps?

Reply:
Sorry, wrong page. I was showing this to someone else during a meeting, and we were looking at Usage Examples. I thought we had buttons there? It has a `modes` pages too: https://pywavelets.readthedocs.io/en/latest/regression/modes.html

Reply:
Ah, the pages under the "Usage examples" section need work to get them working at this time; just the API reference has been implemented. We shall need something like MyST-NB and then convert everything to notebooks, in order for them to run with the `NotebookLite` directive. I reckon Jupytext notebooks will be better than converting to IPyNB. I can get started on this if you would want me to do so?

Reply:
Ah, the pages under the "Usage Examples" section need work to get them working at this time; just the API reference has been implemented. We shall need something like MyST-NB and then convert everything to notebooks in order for them to run with the `NotebookLite` directive. I reckon that Jupytext notebooks will be better than converting to IPyNB. I can get started on this soon if you would want me to do so? We had previously left these for a later time, now sounds like a good time to pick it up.

Reply:
Ah, I misremembered then - that's what I get for doing things during a meeting on a Friday night. 

No need to do it now if MyST-NB isn't ready yet (I'm aware of the work in SciPy on it), but once it's unblocked then yes it would be good to do it.

> I reckon that Jupytext notebooks will be better than converting to IPyNB.

Oh yes, definitely some `.md` format and no `.ipynb`.

Reply:
> Ah, I misremembered then - that's what I get for doing things during a meeting on a Friday night.

No worries! 😄

> No need to do it now if MyST-NB isn't ready yet (I'm aware of the work in SciPy on it), but once it's unblocked then yes it would be good to do it.
>
> Oh yes, definitely some .md format and no .ipynb.

It's ready, actually, it is just SciPy that hits some edge cases because of its custom MathJax, and we managed to find a suitable solution quite recently. The rest of the work there can now be resumed, so nothing remains blocked. I think I will consider that, plus the relatively smaller size of PyWavelets, as my sign and start the work, after which all pages under the PyWavelets documentation will be embellished with interactivity.

Reply:
Sounds great, thanks Agriya.